lldb/source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -731,7 +731,9 @@ bool SwiftUserExpression::Parse(DiagnosticManager &diagnostic_manager,
       if (m_options.GetBindGenericTypes() != lldb::eBindAuto) 
         return false;
       diagnostic_manager.Clear();
-      diagnostic_manager.PutString(eDiagnosticSeverityRemark, "asdfasdf");
+      diagnostic_manager.PutString(eDiagnosticSeverityRemark,
+                                   "Expression evaluation failed. Retrying "
+                                   "without binding generic parameters");
```

Comment:
This is clearly better than before :-) 
I'm just curious: Will the failed expression have produced diagnostics that show how the expression failed?

Reply:
In this case the error would be something like: "Could not realize type of self...", which we don't want to show the user if we're re-running the expression